src/kudu/consensus/quorum_util.cc

```diff
@@ -1010,5 +1010,10 @@ std::string GetQuorumId(const RaftPeerPB& peer, const CommitRulePB& commit_rule)
   return GetQuorumId(peer, IsUseQuorumId(commit_rule));
 }
 
+bool PeerHasNonEmptyQuorumId(const RaftPeerPB &peer) {
```

Comment:
nit: Don't really like the double negative here. `!PeerHasNonEmptyQuorumId` read as "Peer does not have non-empty quorum id".

Maybe just call it PeerHasValidQuorumId, then `!PeerHasValidQuorumId` will read as "Peer does not have valid quorum id".